Polar/inputs-preprocess/subglacial-lakes-2013-2017-resample.py

```diff
@@ -0,0 +1,31 @@
+import xarray as xr
```

Comment:
Due to the preceeding script failing, this is also failing due to missing inputs: 

`15:14 $ python inputs-preprocess/subglacial-lakes-2013-2017-resample.py`
```
Traceback (most recent call last):
  File "/home/conda/deepesdl/envs/cube-gen/lib/python3.9/site-packages/xarray/backends/zarr.py", line 384, in open_group
    zarr_group = zarr.open_consolidated(store, **open_kwargs)
  File "/home/conda/deepesdl/envs/cube-gen/lib/python3.9/site-packages/zarr/convenience.py", line 1187, in open_consolidated
    meta_store = ConsolidatedMetadataStore(store, metadata_key=metadata_key)
  File "/home/conda/deepesdl/envs/cube-gen/lib/python3.9/site-packages/zarr/storage.py", line 2644, in __init__
    meta = json_loads(store[metadata_key])
  File "/home/conda/deepesdl/envs/cube-gen/lib/python3.9/site-packages/zarr/storage.py", line 895, in __getitem__
    raise KeyError(key)
KeyError: '.zmetadata'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/conda/deepesdl/envs/cube-gen/lib/python3.9/site-packages/xarray/backends/zarr.py", line 387, in open_group
    zarr_group = zarr.open_group(store, **open_kwargs)
  File "/home/conda/deepesdl/envs/cube-gen/lib/python3.9/site-packages/zarr/hierarchy.py", line 1182, in open_group
    raise GroupNotFoundError(path)
zarr.errors.GroupNotFoundError: group not found at path ''

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/alicebalfanz/cube-gen/Polar/inputs-preprocess/subglacial-lakes-2013-2017-resample.py", line 25, in <module>
    da = xr.open_zarr(filepath)
  File "/home/conda/deepesdl/envs/cube-gen/lib/python3.9/site-packages/xarray/backends/zarr.py", line 792, in open_zarr
    ds = open_dataset(
  File "/home/conda/deepesdl/envs/cube-gen/lib/python3.9/site-packages/xarray/backends/api.py", line 531, in open_dataset
    backend_ds = backend.open_dataset(
  File "/home/conda/deepesdl/envs/cube-gen/lib/python3.9/site-packages/xarray/backends/zarr.py", line 840, in open_dataset
    store = ZarrStore.open_group(
  File "/home/conda/deepesdl/envs/cube-gen/lib/python3.9/site-packages/xarray/backends/zarr.py", line 404, in open_group
    raise FileNotFoundError(f"No such file or directory: '{store}'")
FileNotFoundError: No such file or directory: '/home/alicebalfanz/data/polar/cubes/repeat_subglacial_lake_drainage_and_filling_thwaites_glacier/thwaites_sublacial_lakes_2013_2017.zarr'
```